experimental/perf/scripts/run_load_tests.py

```diff
@@ -67,15 +170,14 @@ def main():
         "--testcase",
         type=str,
         required=True,
-        help="bench_log | bench_log_scale_step | bench_log_scale_metric",
+        help="log_scalar | log_scalar_scale_step | log_scalar_scale_metric | log_media",
```

Comment:
i think you can use choices here as well

Reply:
done